Tests/SonarScanner.MSBuild.Tasks.UnitTests/SonarScanner.MSBuild.Tasks.UnitTests.csproj

```diff
@@ -53,13 +53,68 @@
       <HintPath>..\..\packages\Microsoft.Build.Utilities.Core.14.3.0\lib\net45\Microsoft.Build.Utilities.Core.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.CodeAnalysis, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
We now need this package to get the _RuleSet_ class, and this package pulls in the others.